packaging/installer/netdata-updater.sh

```diff
@@ -149,6 +149,32 @@ issystemd() {
   return 1
 }
 
+# shellcheck disable=SC2009
+running_under_anacron() {
+    if [ "$(uname -s)" = "Linux" ] && [ -r "/proc/$$/stat" ]; then
+        ppid="$(cut -f 4 -d ' ' "/proc/$$/stat")"
+        if [ -n "${ppid}" ] && [ -r "/proc/${ppid}/comm" ]; then
+            grep -q anacron "/proc/${ppid}/comm" && return 0
+
+            ppid2="$(cut -f 4 -d ' ' "/proc/${ppid}/stat")"
+
+            if [ -n "${ppid2}" ] && [ -r "/proc/${ppid2}/comm" ]; then
+                grep -q anacron "/proc/${ppid2}/comm" 2>/dev/null && return 0
+            fi
+        fi
+    else
+        ppid="$(ps -o pid= -o ppid= 2>/dev/null | grep -e "^ *$$" | xargs | cut -f 2 -d ' ')"
+        if [ -n "${ppid}" ]; then
+            ps -o pid= -o command= 2>/dev/null | grep -e "^ ${ppid}" | grep -q anacron && return 0
```

Comment:
```suggestion
            ps -o pid= -o command= 2>/dev/null | grep -e "^ ${ppid}" | grep -q anacron && return 0
```

Are you sure it is always one here? Or forgot to change to `"^ *${ppid}"`?

Reply:
You’re right, I missed this one somehow.